### DISCUSSION TITLE
The Founder’s Club - SC, webpage and token sale

### DISCUSSION THREAD

**Discussion by paulgosnell** [fitlink-app/fitlink-monorepo#270] [Ideas]
We need a new SC, which will be used for a $BFIT token sale where we sell $BFIT at a 20% discount to the target listing price of $0.20, so the sale price will be $0.16 for 1 $BFIT

All parameters to include in the SC;

- Sale start: asap
- Sale 1 duration: 10 days
- Sale 2 duration: 14 days
- Sale 3 duration: 21 days
- Sale 1 amount: 937538 (25%)
- Sale 2 amount: 1312553 (35%)
- Sale 3 amount: 1500060 (40%)
- Sale 1 price: $0.11 (-45%)
- Sale 2 price: $0.14 (-30%)
- Sale 3 price: $0.15 (-25%)
- Vesting period: ~12~ 6 months after TGE
- Minimum buy amount: $300
- Maximum buy amount: $3000
- Purchase with: $USK, $UST, $ATOM
- TGE: TBC

Other functions needed;

- Admin can access and withdraw
- The ability to show how much a holder holds inside app

For context, this SC will be for a group of users we'll call the **Founders Club** 

It's more from a strategic marketing PoV where we want users, early adopters, to feel they're getting more from BFIT than just a token sale; we want them to feel part of the project and to have some ownership. We have therefore created a Founder’s Club where members get lifelong member benefits as long as they maintain a minimum stake in BFIT.

The Founder’s Club benefits include;

1. Early access to a BFIT sale page granting members an exclusive price of $.16 per $BFIT
2. A minimum amount of $300 worth of $BFIT per member
3. A percentage of all future buy and sell taxes is shared with the club
4. An exclusive member profile badge
5. Direct access to the founding product team, exclusive email updates for members, and private Telegram/Discord channels
6. Early access to all future feature releases
7. Free merchandise, including Hoodies, T-’s and beanies, every quarter
8. A chance to get involved in shaping the product roadmap

The new webpage will look like the below screenshot, and the 'apply here' button will take them to a TypeForm where we'll ask questions like;

1. Do you exercise
2. What is your exercise
3. How often do you exercise
4. Do you track your exercise
5. What app or wearable do you track with
6. Are you iOS and Android
7. Do you use other M2E apps
8. What rewards would you go for
9. Would you like this at your workplace
10. Would you recommend/invite others

And the reason for the questions is to ensure we're getting potential users of the app rather than people that will buy and dump, although the vesting schedule should help fix any dumping problems.

User flow;

1. Select the amount of BFIT
2. Connect their wallet
3. Complete KYC
4. Purchase BFIT using USK
5. Confirmation of purchase
5. See in BFIT App their 'Founders Club' badge and the amount of BFIT bought (in BFIT wallet)
6. Earn more BFIT whilst using the beta app
7. All bought and earned BFIT get combined
8. At TGE, all BFIT gets minted and claimed

![Founders Club](https://user-images.githubusercontent.com/615605/212557161-8f09eda2-d0e5-47f3-935b-d1d2d1fb9c53.png)

**Comment by paulgosnell** [comment]
hey @Clish254 we ok to create a SC to cover the above?

**Reply by Clish254** [reply]
Yes

**Comment by paulgosnell** [comment]
Founders Club website and TypeForm built and live > https://bfitcoin.com/founders-club
